common/build.rs

```diff
@@ -1,4 +1,5 @@
 fn main() -> Result<(), Box<dyn std::error::Error>> {
+    println!("Running build.rs");
```

Comment:
```suggestion
```
stdout in build.rs is used to configure cargo: https://doc.rust-lang.org/cargo/reference/build-scripts.html#outputs-of-the-build-script

So this won't be displayed in the TTY, and while likely benign, I would remove it